src/stores/orderStore.ts

```diff
@@ -51,92 +52,101 @@ class OrderStore extends BaseStore {
   placeOrder = async (orderType: string, body: any) => {
     switch (orderType) {
       case OrderType.Market:
-        return this.api
-          .placeMarket(body)
-          .then(this.orderPlacedSuccessfully, this.orderPlacedUnsuccessfully)
-          .then(() => Promise.resolve());
+        return (
+          this.api
+            .placeMarket(body)
+            // tslint:disable-next-line:no-empty
+            .then(() => {}, this.orderPlacedUnsuccessfully)
+            .then(() => Promise.resolve())
```

Comment:
why resolve in the next step?

will `then(Promise.resolve, this.orderPlacedUnsuccessfully)` work?